_includes/head.html

```diff
@@ -0,0 +1,62 @@
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{{ page.title }} - Bazel</title>
+
+    <script>
+      var current_url = window.location.href;
```

Comment:
Removed all the redirect code except for the http://blog.bazel.build -> https://blog.bazel.build redirect.